extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -639,10 +664,15 @@ export class AmpA4A extends AMP.BaseElement {
     if (!this.adPromise_) {
       return Promise.resolve();
     }
+    const layoutCallbackStart = this.getNow_();
     // Promise chain will have determined if creative is valid AMP.
     return this.adPromise_.then(creativeMetaData => {
+      const delta = this.getNow_() - layoutCallbackStart;
+      this.protectedEmitLifecycleEvent_('adPromiseChainDelay', {
```

Comment:
The name here is big vague... perhaps it should "layout" (e.g. layoutAdPromiseDelay)?